Libraries/Text/RCTTextView.m

```diff
@@ -127,6 +128,7 @@ - (NSString *)text
 
 - (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range replacementText:(NSString *)text
 {
+  [self sendKeyValueForText:text];
```

Comment:
This does weird things with paste and auto-correct type stuff, sending full strings of characters.